AnkiDroid/src/main/java/com/ichi2/anki/DeckPicker.kt

```diff
@@ -303,7 +303,9 @@ open class DeckPicker :
         ActivityResultContracts.StartActivityForResult(),
         DeckPickerActivityResultCallback {
             if (it.resultCode == RESULT_OK) {
-                onSelectedPackageToImport(it.data!!)
+                lifecycleScope.launch(Dispatchers.IO) {
+                    onSelectedPackageToImport(it.data!!)
+                }
```

Comment:
we have a helper for catching errors (which would be nice if it accepted a `CoroutineContext` param like `launchCatching` does)

```suggestion
                launchCatchingTask {
                    withContext(Dispatchers.IO) {
                        onSelectedPackageToImport(it.data!!)   
                    }
                }
```